core/math/basis.cpp

```diff
@@ -988,14 +988,14 @@ void Basis::rotate_sh(real_t *p_values) {
 	// http://filmicworlds.com/blog/simple-and-fast-spherical-harmonic-rotation/
 	// this code is Public Domain
 
-	const static real_t s_c3 = 0.94617469575; // (3*sqrt(5))/(4*sqrt(pi))
-	const static real_t s_c4 = -0.31539156525; // (-sqrt(5))/(4*sqrt(pi))
-	const static real_t s_c5 = 0.54627421529; // (sqrt(15))/(4*sqrt(pi))
+	const static real_t s_c3 = 0.946'174'695'75; // (3*sqrt(5))/(4*sqrt(pi))
+	const static real_t s_c4 = -0.315'391'565'25; // (-sqrt(5))/(4*sqrt(pi))
+	const static real_t s_c5 = 0.546'274'215'29; // (sqrt(15))/(4*sqrt(pi))
 
-	const static real_t s_c_scale = 1.0 / 0.91529123286551084;
-	const static real_t s_c_scale_inv = 0.91529123286551084;
+	const static real_t s_c_scale = 1.0 / 0.915'291'232'865'510'84;
+	const static real_t s_c_scale_inv = 0.915'291'232'865'510'84;
 
-	const static real_t s_rc2 = 1.5853309190550713 * s_c_scale;
+	const static real_t s_rc2 = 1.585'330'919'055'071'3 * s_c_scale;
```

Comment:
It looks pretty weird to me on arbitrary decimal numbers.

IMO using number separators makes sense for things like `10'000'000`, but for the arbitrary result of a computation, I don't think it's useful. This is copy pasted output from a calculator/terminal and the only way to proofread it is to just copy paste a new value on top and check the diff. It's not something that you have to read by intermediate units.